cypress/integration/Images/LeadImageTest/index.js

```diff
@@ -0,0 +1,22 @@
+// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Then('lead image is visible', () => {
+    cy.get('.centered-set.image-medium').should('be.visible');
+});
+
+And('{string} caption is correct', (caption) => {
+    cy.get('.caption-container').should('contain.text', caption);
```

Comment:
this locator should be related to the lead image tag - otherwise it will pick up any other image caption on the page

Reply:
fixed this